tools/twix/src/panels/manual_camera_calibration.rs

```diff
@@ -0,0 +1,207 @@
+use eframe::egui::{Response, Slider, Ui, Widget};
+use log::{error, info};
+use serde_json::Value;
+use std::{ops::RangeInclusive, sync::Arc};
+use tokio::sync::mpsc;
+use types::configuration::CameraMatrixParameters;
+
+use crate::{
+    nao::Nao, panel::Panel, repository_parameters::RepositoryParameters, value_buffer::ValueBuffer,
+};
+
+use super::parameter::{add_save_button, subscribe};
```

Comment:
Yes that was also my justification for adding the dependency since manual calibration is a specialization or in another framework might be an inheritance case.
Also I didn't want to have code duplication when both should do exactly the same.